### ISSUE TITLE
error: template "xxx" not exists or not selected in test suite when running on Windows 10 with Gitbash

### ISSUE THREAD

**Issue by igdianov** [lrills/helm-unittest#55]
Running `heml unittest .` from `__fixtures__/basic` folder using `Gitbash` terminal does not work on Windows 10 due to Windows file separators being used to read templates from test suite manifest: 

```sh
User@pc MINGW64 ~/git/helm-unittest/__fixtures__/basic (master): helm unittest .
....
 FAIL  test service     tests\service_test.yaml
        - should pass

                - asserts[0] `contains` fail

                        Error:
                                template "basic\templates\service.yaml" not exists or not selected in test suite

                - asserts[1] `equal` fail

                        Error:
                                template "basic\templates\service.yaml" not exists or not selected in test suite

                - asserts[2] `equal` fail

                        Error:
                                template "basic\templates\service.yaml" not exists or not selected in test suite

        - should render right if values given

                - asserts[0] `contains` fail

                        Error:
                                template "basic\templates\service.yaml" not exists or not selected in test suite

                - asserts[1] `equal` fail

                        Error:
                                template "basic\templates\service.yaml" not exists or not selected in test suite


Charts:      1 failed, 0 passed, 1 total
Test Suites: 3 failed, 0 passed, 3 total
Tests:       7 failed, 0 passed, 7 total
Snapshot:    0 passed, 0 total
Time:        119.1127ms

Error: plugin "unittest" exited with error
```

**Comment by libkad** [comment]
I have the same issue

**Comment by lrills** [comment]
It should be fixed in 0.1.4, please `helm plugin remove unittest` and install again (`helm plugin update` not work before 0.1.4)
